src/builder.js

```diff
@@ -354,7 +354,11 @@ export function sfBuilderService() {
         form['choices'] = {titleMap: parameter.choices.value};
 
         if (parameter.choices.details && parameter.choices.details.key_reference) {
-          let field = parentKey + '.' + parameter.choices.details.key_reference;
+          let field = parameter.choices.details.key_reference;
+
+          if (field !== 'instance_name') {
```

Comment:
This is duplicated in a lot of places. If we wanted to enable `system_version` or anything else that isn't a true parameter, we'd have to duplicate this logic again. 

Recommend pulling this into a function and calling the function.